src/main/java/com/iwaconsolti/league/manager/controller/PopulatedLeaguesController.java

```diff
@@ -0,0 +1,86 @@
+package com.iwaconsolti.league.manager.controller;
+
+import com.iwaconsolti.league.manager.model.Players;
+import com.iwaconsolti.league.manager.model.Teams;
+import com.iwaconsolti.league.manager.service.ILeagues;
+import jakarta.annotation.PostConstruct;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.context.annotation.Profile;
+import org.springframework.web.bind.annotation.*;
+
+@Profile("populated")
+@RestController
```

Comment:
This should not be a RestController, it should only be a simple Component type. Since it doesnt have any endpoint declared only the posconstruc